stacker/tests/config/translators/test_file.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import mock
+import base64
+import troposphere
+
+from stacker.config.translators.file import parameterized_codec, get_file_value
+
+
+class TestFileTranslator(unittest.TestCase):
+    def setUp(self):
```

Comment:
Ha, oops. I thought I'd use it, so I put it there, and never removed it when it went unused.